sec_certs/sample/common_criteria.py

```diff
@@ -190,6 +190,7 @@ class CCHeuristics(ComplexSerializableType):
         related_cves: Optional[Set[str]] = field(default=None)
         cert_lab: Optional[List[str]] = field(default=None)
         cert_id: Optional[str] = field(default=None)
```

Comment:
Eh, changing the format like this creates issues for the site and all downstream users. I understand it is very much alpha stage software right now, but perhaps we can just replace the `cert_id` value with the new normalized one. It was a result of a heuristic applied to the cert ID matches anyway, so now it will just be a better/normalized heuristic, so that the field name can stay. One can always look at the `rules_cert_id` matches in the target or report and get additional info there.

Reply:
Yes of course, @GeorgeFI, can we discard `normalized_cert_id` a keep `cert_id` in `CCHeuristics` object? Doing otherwise would break the webpage.